Dockerfile

```diff
@@ -1,40 +1,50 @@
 ARG BUILD_FROM
 FROM $BUILD_FROM
 
-# Set shell
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Setup locals
-RUN apt-get update && apt-get install -y --no-install-recommends \
-        jq \
-        git \
-        python3-setuptools \
-    && rm -rf /var/lib/apt/lists/* \
-ENV LANG C.UTF-8
-
-# Install docker
-# https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/
-RUN apt-get update && apt-get install -y --no-install-recommends \
-        apt-transport-https \
-        ca-certificates \
-        curl \
-        software-properties-common \
-        gpg-agent \
-    && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - \
-    && add-apt-repository "deb https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" \
-    && apt-get update && apt-get install -y --no-install-recommends \
-        docker-ce \
-        docker-ce-cli \
-        containerd.io \
-    && rm -rf /var/lib/apt/lists/*
+ENV \
+    VCN_OTP_EMPTY=true \
+    LANG=C.UTF-8
 
-# Setup arm binary support
 ARG BUILD_ARCH
-RUN if [ "$BUILD_ARCH" != "amd64" ]; then exit 0; else \
-    apt-get update && apt-get install -y --no-install-recommends \
-        qemu-user-static \
-        binfmt-support \
-    && rm -rf /var/lib/apt/lists/*; fi
+ARG VCN_VERSION
+
+RUN \
+    set -x \
+    && apk add --no-cache \
+        git \
+        docker \
+    && apk add --no-cache --virtual .build-dependencies \
+        build-base \
+        go \
+    \
+    && git clone -b v${VCN_VERSION} --depth 1 \
+        https://github.com/codenotary/vcn \
+    && cd vcn \
+    \
+    # Fix: https://github.com/codenotary/vcn/issues/131
+    && go get github.com/codenotary/immudb@4cf9e2ae06ac2e6ec98a60364c3de3eab5524757 \
```

Comment:
Should we store this SHA as a build arg?

Reply:
It's part of the workaround from above, it has no future, so would avoid.